frontend/javascripts/test/snapshots/public-test/test-bundle/test/backend-snapshot-tests/annotations.e2e.js.md

```diff
@@ -461,6 +464,8 @@ Generated by [AVA](https://avajs.dev).
         created: 'created',
         creationInfo: null,
         dataSet: '2012-06-28_Cortex',
+        datasetId: '570b9f4e4bb848d0885ee711',
+        datasetName: '2012-06-28_Cortex',
         editPosition: [
```

Comment:
@fm3 I just noticed by check which files where updated. I thought we wanted to avoid having `datasetName` and `dataSet` in parallel? 

Although checking: https://github.com/scalableminds/webknossos/pull/8075#discussion_r1827721741 shows that we did not clearly have this decision. Should I include a legacy route for anything that returns / receives a task? There is already a legacy route for creating and updating a task, but the publicWrites in `TaskService` still returns 
```
 "dataSet" -> dataset.name,
 "datasetName" -> dataset.name,
 "datasetId" -> dataset._id, // Only used for csv serialization in frontend.
```
Should I add a legacy route for this to not always send dataSet and datasetName?

Reply:
This can also be seen in `tasks.e2e.js.md`